src/dispatching/update_listeners.rs

```diff
@@ -116,11 +116,11 @@ pub trait UpdateListener<E>: Stream<Item = Result<Update, E>> {
 }
 impl<S, E> UpdateListener<E> for S where S: Stream<Item = Result<Update, E>> {}
 
-/// Returns a long polling update listener with the default configuration.
+/// Returns a long polling update listener with `timeout` of 1 minute.
 ///
 /// See also: [`polling`](polling).
 pub fn polling_default(bot: Arc<Bot>) -> impl UpdateListener<RequestError> {
-    polling(bot, None, None, None)
+    polling(bot, Some(Duration::from_secs(1)), None, None)
```

Comment:
This seems wrong. The doc says 1min not 1sec.

Reply:
Fixed.